hedera-node/hedera-util-service-impl/src/main/java/com/hedera/node/app/service/util/impl/handlers/AtomicBatchHandler.java

```diff
@@ -51,6 +52,8 @@
 public class AtomicBatchHandler implements TransactionHandler {
     private final Function<Transaction, TransactionBody> bodyParser;
```

Comment:
You could probably use similar pattern to get dispatcher and use in pure checks. You don't need the `PureChecksContext`

Reply:
But since we already made the change, thats okay to leave. We can just remove `Configuration` from `pureChecksContext`